Ninjathulhu/Ninjathulhu.Game/Component.cs

```diff
@@ -1,16 +1,13 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
+﻿using System.Collections.Generic;
 
 namespace Ninjathulhu.Game
 {
-    class ComponentProperties
+    public class ComponentProperties
```

Comment:
Making these public so they're visible from the test project. I suppose an InternalsVisibleTo attribute would be safer.